jest.config.js

```diff
@@ -58,4 +58,8 @@ module.exports = {
   // Log the test results with dynamic Loki tags. Drone CI only
   reporters: ['default', ['<rootDir>/public/test/log-reporter.js', { enable: process.env.DRONE === 'true' }]],
   watchPlugins: ['jest-watch-typeahead/filename', 'jest-watch-typeahead/testname'],
+  modulePathIgnorePatterns: [
+    // Avoid the `jest-haste-map: duplicate manual mock found` warning
+    ".*__mocks__.*"
```

Comment:
I guess this isn't an issue unless someone wants to require anything inside a `__mocks__` directory?